### PR TITLE
babel-preset-env and babel-polyfill

### PR DESCRIPTION
**Pull request by jp-bbc**
_babel-preset-env and -polyfill builds in Razzle. Also installed babel-preset-razzle. .babelrc features setup for preset-env and polyfill browser targets._

* [x] Fixes https://github.com/bbc/simorgh/issues/255
* [ ] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

**Review by jamesbhobbs — DISMISSED** [review]
This shouldn't be in CR, the tests don't pass.

Based on a 2 second look at the tests you're adding something to build that isn't a production dependency which therefore breaks the production build.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "presets": [
+        ["env", {
+            "entry": ["babel-polyfill", "./src/client.js"],
```
Comment (jtart) [inline .babelrc]: This looks like webpack config and should be removed.

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                    "edge >= 37",
+                    "safari >= 9",
+                    "opera >= 40",
+                    "op_mini >= 18",
+                    "Android >= 7",
+                    "and_chr >= 53",
+                    "and_ff >= 49",
+                    "ios_saf >= 10"]
+            }
+        }], ["razzle"]
```
Comment (jtart) [inline .babelrc]: This isn't how we add the razzle babel config, need to do something [like this](https://github.com/jaredpalmer/razzle/#customizing-babel-config).

File: package.json [hunk 3]
```diff
@@ -64,6 +66,7 @@
     "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.5",
     "babel-loader": "^7.1.4",
+    "babel-preset-razzle": "^2.0.0",
```
Comment (jtart) [inline package.json]: You don't need to add this.

**Review by jtart — CHANGES_REQUESTED** [review]
You also have conflicts in your webpack bundle files.

**Review by jtart — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

**Pull request merged**